### ISSUE TITLE
[1.20.1] Render Crash Loop between Sophisticated Storage

### ISSUE THREAD

**Issue by whirlwind0204** [Asek3/Rubidium#586]
## Issue: 
**Game crashes when I log into my server due to a rendering issue between Rubidium and the Limited Barrel from Sophisticated Storage**
## Versions:
- Rubidium - 0.6.5 (Due to issues with later versions)
- Sophisticated Storage - 0.8.46.579
- Soph. Core - 0.5.83.395
- A number of other mods listed in crash report
## Crash Report:
- https://pastebin.com/6ucDA2e9 - _From a few days ago; the one P3pp3rF1y was sent_
- https://pastebin.com/Y166tvuM - _From today, in case it includes any more useful info_

## Relevant Info:
I contacted the Sophisticated Storage dev and posted a bug report because I assumed it was a problem on their side, this was their response:

> **P3pp3rF1y** —
I haven't seen this one yet, but that is for Rubidium to figure out as it crashes in their code and I am not aware of what they are doing there. Also limited barrels definitely work with Rubidium (I test them in my pack / dev environment) so this is more than just a combination of Rubidium + limited barrels"

Additionally, it seems players who aren't nearby are able to play without issue and the server also runs fine.

Any potential fixes would be greatly appreciated

**Comment by whirlwind0204** [comment]
## New Info: 
My friend managed to break the problem block, but unfortunately it's now crashing from another block; one that had no issue before and has been placed a while

## Crash Report:
https://pastebin.com/xNMjwcxp

**Comment by lewxdev** [comment]
I'm also having the same issue when rendering sophisticated storage block entities
Here's my crash report: https://mclo.gs/Jsucntn

**Comment by whirlwind0204** [comment]
> I'm also having the same issue when rendering sophisticated storage block entities Here's my crash report: https://mclo.gs/Jsucntn

Do you also have trouble with versions 7.0.0 onwards?
When i try to launch with those versions the launching process immediately cuts out without a crash report

**Comment by lewxdev** [comment]
> Do you also have trouble with versions 7.0.0 onwards? When i try to launch with those versions the launching process immediately cuts out without a crash report

Yeah, I'm using 0.6.5 because there's a known incompatibility with version 0.7.0
- https://github.com/Asek3/Rubidium/issues/587
- https://github.com/Asek3/Oculus/issues/430
- https://github.com/Asek3/Oculus/issues/434
